teamcity: tests timing out with no log output

The following tests appear to have failed:

[#142484](https://teamcity.cockroachdb.com/viewLog.html?buildId=142484):

```
--- FAIL: test (proposer evaluated kv)/TestParseSentinelValueWithNewColumnInSentinelFamily (0.038s)
Test ended in panic.
panic: test timed out after 2m0s

goroutine 216669 [running]:
panic(0x1854c00, 0xc422640100)
	/usr/local/go/src/runtime/panic.go:500 +0x1a1
testing.startAlarm.func1()
	/usr/local/go/src/testing/testing.go:918 +0x10b
created by time.goFunc
	/usr/local/go/src/time/sleep.go:154 +0x44

goroutine 1 [chan receive]:
testing.(*T).Run(0xc4202de3c0, 0x1a88566, 0x33, 0x1c572f0, 0xc4200e7c01)
	/usr/local/go/src/testing/testing.go:647 +0x316
testing.RunTests.func1(0xc4202de3c0)
	/usr/local/go/src/testing/testing.go:793 +0x6d
testing.tRunner(0xc4202de3c0, 0xc42018ce00)
	/usr/local/go/src/testing/testing.go:610 +0x81
testing.RunTests(0x1c58e08, 0x2627b40, 0x79, 0x79, 0x2b51db8)
	/usr/local/go/src/testing/testing.go:799 +0x319
testing.(*M).Run(0xc42018cef8, 0x2b518e0)
	/usr/local/go/src/testing/testing.go:743 +0x85
github.com/cockroachdb/cockroach/pkg/sql_test.TestMain(0xc42018cef8)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/main_test.go:227 +0xbd
main.main()
	github.com/cockroachdb/cockroach/pkg/sql/_test/_testmain.go:642 +0xc6

goroutine 17 [syscall, 1 minutes, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:2086 +0x1

goroutine 20 [chan receive]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).flushDaemon(0x2b16040)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1012 +0x77
created by github.com/cockroachdb/cockroach/pkg/util/log.init.1
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:583 +0xc4

goroutine 21 [chan receive]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0x2b16040)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1039 +0x5b
created by github.com/cockroachdb/cockroach/pkg/util/log.init.1
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:584 +0xe8

goroutine 216752 [runnable]:
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc422546ff0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:206
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216731 [semacquire]:
sync.runtime_notifyListWait(0xc4225efd10, 0xc400000003)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc4225efd00)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc422055600, 0xc422ba02d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:205 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:174 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc42309bf80)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216751 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).redirectOnOrAcquireLease.func2(0xc421ebfb50, 0xc422990360, 0xc421b98c00, 0x2b23c00d14c0, 0xc4227a65a0, 0xc421ebfb58, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:1052 +0x520
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).redirectOnOrAcquireLease(0xc421b98c00, 0x2b23c00d14c0, 0xc4227a65a0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x653f62)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:1102 +0x177
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).getLeaseForGossip.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:4273 +0x72
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTask(0xc422ba02d0, 0xc421ebfcd0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:239 +0x105
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).getLeaseForGossip(0xc421b98c00, 0x2b23c00d14c0, 0xc4227a65a0, 0xe3aec0, 0xc421657d28)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:4304 +0x131
github.com/cockroachdb/cockroach/pkg/storage.(*Store).startGossip.func1(0x2b23c00d14c0, 0xc4227a65a0, 0xc421b98c00, 0x8, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:1278 +0x3f
github.com/cockroachdb/cockroach/pkg/storage.(*Store).startGossip.func4()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:1336 +0x2ba
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc422546fa0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216749 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*Store).Start.func3()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:1220 +0x165
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc421cb4280)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216738 [semacquire]:
sync.runtime_notifyListWait(0xc4225efd10, 0xc40000000a)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc4225efd00)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc422055600, 0xc422ba02d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:205 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:174 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc421cb40e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216732 [semacquire]:
sync.runtime_notifyListWait(0xc4225efd10, 0xc400000004)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc4225efd00)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc422055600, 0xc422ba02d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:205 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:174 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc42309bfc0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216704 [chan receive]:
github.com/cockroachdb/cockroach/pkg/storage/engine.(*RocksDB).open.func1(0xc422d1d860)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/engine/rocksdb.go:382 +0x44
created by github.com/cockroachdb/cockroach/pkg/storage/engine.(*RocksDB).open
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/engine/rocksdb.go:383 +0x702

goroutine 216703 [select]:
github.com/cockroachdb/cockroach/pkg/gossip.(*Gossip).manage.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/gossip/gossip.go:1075 +0x399
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc422d06690)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216737 [runnable]:
sync.runtime_notifyListWait(0xc4225efd10, 0xc400000000)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc4225efd00)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc422055600, 0xc422ba02d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:205 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:174 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc421cb4080)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216741 [semacquire]:
sync.runtime_notifyListWait(0xc4225efd10, 0xc40000000d)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc4225efd00)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc422055600, 0xc422ba02d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:205 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:174 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc421cb4180)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216750 [runnable]:
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc422546f50)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:206
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216730 [semacquire]:
sync.runtime_notifyListWait(0xc4225efd10, 0xc400000002)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc4225efd00)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc422055600, 0xc422ba02d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:205 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:174 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc42309bf60)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216695 [IO wait]:
net.runtime_pollWait(0x2b23c0c1a048, 0x72, 0x0)
	/usr/local/go/src/runtime/netpoll.go:160 +0x59
net.(*pollDesc).wait(0xc42028ff00, 0x72, 0xc420672608, 0xc4200900b0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x38
net.(*pollDesc).waitRead(0xc42028ff00, 0x2635ba0, 0xc4200900b0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x34
net.(*netFD).accept(0xc42028fea0, 0x0, 0x2632ce0, 0xc421c88060)
	/usr/local/go/src/net/fd_unix.go:419 +0x238
net.(*TCPListener).accept(0xc420712110, 0xc4206726f0, 0xc420672700, 0x67ec20)
	/usr/local/go/src/net/tcpsock_posix.go:132 +0x2e
net.(*TCPListener).Accept(0xc420712110, 0x1c57c68, 0xc421f52540, 0xc4226400e0, 0xc422c3fae0)
	/usr/local/go/src/net/tcpsock.go:222 +0x49
github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux.(*cMux).Serve(0xc421f52540, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux/cmux.go:124 +0xa4
github.com/cockroachdb/cockroach/pkg/server.(*Server).Start.func3()
	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:490 +0x2f
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc4204b1260)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216690 [select]:
github.com/cockroachdb/cockroach/pkg/kv.(*TxnCoordSender).printStatsLoop(0xc4220542c0, 0x2b23c00d14c0, 0xc42306e1b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/txn_coord_sender.go:209 +0xbdc
github.com/cockroachdb/cockroach/pkg/kv.NewTxnCoordSender.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/txn_coord_sender.go:193 +0x98
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc4224a9b20)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216694 [chan receive]:
github.com/cockroachdb/cockroach/pkg/server.(*Server).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:478 +0x71
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc42300a750)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216620 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).tryAddWriteCmd(0xc421b98c00, 0x2b23c00d14c0, 0xc423002030, 0x14a10cec33c6f265, 0x0, 0x0, 0x0, 0x1, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:2165 +0xba4
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).addWriteCmd(0xc421b98c00, 0x2b23c00d14c0, 0xc423002030, 0x14a10cec33c6f265, 0x0, 0x0, 0x0, 0x1, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:1984 +0x90
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).Send(0xc421b98c00, 0x2b23c00d14c0, 0xc423002030, 0x14a10cec33c6f265, 0x0, 0x0, 0x0, 0x1, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:1391 +0x226
github.com/cockroachdb/cockroach/pkg/storage.(*pendingLeaseRequest).requestLeaseAsync.func1(0x2b23c00d14c0, 0xc4227a6930)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica_range_lease.go:206 +0x630
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc422ba02d0, 0x2304859, 0x1e, 0xe7, 0x0, 0x0, 0xc42250e000, 0x2b23c00d0460, 0xc420090ca8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:283 +0xdf
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:284 +0x14a

goroutine 216740 [semacquire]:
sync.runtime_notifyListWait(0xc4225efd10, 0xc40000000c)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc4225efd00)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc422055600, 0xc422ba02d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:205 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:174 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc421cb4140)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216701 [chan receive]:
github.com/cockroachdb/cockroach/pkg/gossip.(*server).start.func3()
	/go/src/github.com/cockroachdb/cockroach/pkg/gossip/server.go:366 +0x66
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc4204b13e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216691 [select]:
github.com/cockroachdb/cockroach/pkg/sql.(*LeaseManager).RefreshLeases.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/lease.go:1204 +0xc76
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc42306f2f0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216698 [chan receive]:
github.com/cockroachdb/cockroach/pkg/server.(*Server).Start.func6()
	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:505 +0x88
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc421f52e00)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216736 [semacquire]:
sync.runtime_notifyListWait(0xc4225efd10, 0xc400000008)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc4225efd00)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc422055600, 0xc422ba02d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:205 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:174 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc421cb4060)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216739 [semacquire]:
sync.runtime_notifyListWait(0xc4225efd10, 0xc40000000b)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc4225efd00)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc422055600, 0xc422ba02d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:205 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:174 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc421cb4120)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216744 [semacquire]:
sync.runtime_notifyListWait(0xc4225efd10, 0xc400000009)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc4225efd00)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc422055600, 0xc422ba02d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:205 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:174 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc421cb41e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216728 [chan receive]:
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:164 +0x55
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc42309bf20)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216733 [semacquire]:
sync.runtime_notifyListWait(0xc4225efd10, 0xc400000005)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc4225efd00)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc422055600, 0xc422ba02d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:205 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:174 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc42309bfe0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216702 [select]:
github.com/cockroachdb/cockroach/pkg/gossip.(*Gossip).bootstrap.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/gossip/gossip.go:1037 +0x4de
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc422d06680)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216700 [select]:
github.com/cockroachdb/cockroach/pkg/server.(*Server).Start.func8()
	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:525 +0x295
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc42300a7b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216729 [semacquire]:
sync.runtime_notifyListWait(0xc4225efd10, 0xc400000001)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc4225efd00)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc422055600, 0xc422ba02d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:205 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:174 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc42309bf40)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216697 [chan receive]:
github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux.(*muxListener).Accept(0xc4204b1220, 0xc420712110, 0xc422630060, 0x18a4b40, 0x2611910)
	<autogenerated>:9 +0x7f
crypto/tls.(*listener).Accept(0xc4204b12a0, 0xc422630030, 0x18a4b40, 0x2611910, 0x19451c0)
	/usr/local/go/src/crypto/tls/tls.go:52 +0x37
net/http.(*Server).Serve(0xc4204bfa80, 0x263aae0, 0xc4204b12a0, 0x0, 0x0)
	/usr/local/go/src/net/http/server.go:2273 +0x1ce
github.com/cockroachdb/cockroach/pkg/server.(*Server).Start.func5()
	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:501 +0x42
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc4204b12c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216699 [chan receive]:
github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux.(*muxListener).Accept(0xc4204b1120, 0x1c58b00, 0xc4224d0c00, 0x2642e20, 0xc4204b1120)
	<autogenerated>:9 +0x7f
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).Serve(0xc4224d0c00, 0x2642e20, 0xc4204b1120, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:369 +0x187
github.com/cockroachdb/cockroach/pkg/server.(*Server).Start.func7()
	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:516 +0x43
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc4204b1360)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216734 [semacquire]:
sync.runtime_notifyListWait(0xc4225efd10, 0xc400000006)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc4225efd00)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc422055600, 0xc422ba02d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:205 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:174 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc421cb4000)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216696 [chan receive]:
github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux.(*muxListener).Accept(0xc4204b1200, 0xc421b0c180, 0x18a4b40, 0x2611910, 0x19451c0)
	<autogenerated>:9 +0x7f
net/http.(*Server).Serve(0xc4204bfb80, 0x2642e20, 0xc4204b1200, 0x0, 0x0)
	/usr/local/go/src/net/http/server.go:2273 +0x1ce
github.com/cockroachdb/cockroach/pkg/server.(*Server).Start.func4()
	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:494 +0x3c
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc4204b1280)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216754 [runnable]:
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc422ba02d0, 0x2308783, 0x13, 0x12d, 0x0, 0x0, 0xc4207e2cc0, 0x2b23c00d0460, 0xc420090ca0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:279
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:284 +0x14a

goroutine 216692 [chan receive]:
github.com/cockroachdb/cockroach/pkg/util/netutil.MakeServer.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/util/netutil/net.go:98 +0x70
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc4204b0f80)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216743 [semacquire]:
sync.runtime_notifyListWait(0xc4225efd10, 0xc40000000f)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc4225efd00)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc422055600, 0xc422ba02d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:205 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:174 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc421cb41c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216735 [semacquire]:
sync.runtime_notifyListWait(0xc4225efd10, 0xc400000007)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc4225efd00)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc422055600, 0xc422ba02d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:205 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:174 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc421cb4040)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216753 [runnable]:
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc4207e2a20)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:206
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216746 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*Store).raftTickLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:3410 +0x334
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc4207e29c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216742 [semacquire]:
sync.runtime_notifyListWait(0xc4225efd10, 0xc40000000e)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc4225efd00)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc422055600, 0xc422ba02d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:205 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:174 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc421cb41a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216745 [semacquire]:
sync.runtime_Semacquire(0xc4220556a4)
	/usr/local/go/src/runtime/sema.go:47 +0x30
sync.(*WaitGroup).Wait(0xc422055698)
	/usr/local/go/src/sync/waitgroup.go:131 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Wait(0xc422055600)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:180 +0x34
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Wait-fm()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:3393 +0x2a
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc4207e29b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216693 [chan receive]:
github.com/cockroachdb/cockroach/pkg/util/netutil.MakeServer.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/util/netutil/net.go:98 +0x70
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc4204b0fc0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216353 [chan receive]:
github.com/cockroachdb/cockroach/pkg/rpc.NewContext.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:136 +0x76
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc4204b0420)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66

goroutine 216352 [runnable]:
reflect.Value.Field(0x19180c0, 0xc4227a7260, 0x199, 0x0, 0x183b300, 0xc4227a72c0, 0x197)
	/usr/local/go/src/reflect/value.go:739
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/proto.mergeStruct(0x19180c0, 0xc4227a72c0, 0x199, 0x19180c0, 0xc4227a7260, 0x199)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/proto/clone.go:84 +0x216
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/proto.Clone(0x263aca0, 0xc4227a7260, 0x11, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/proto/clone.go:52 +0x1a2
github.com/cockroachdb/cockroach/pkg/util/protoutil.Clone(0x263aca0, 0xc4227a7260, 0xc420090ca8, 0x2b23c00d14c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/protoutil/clone.go:64 +0x122
github.com/cockroachdb/cockroach/pkg/storage.(*Stores).updateBootstrapInfo(0xc42028fdc0, 0xc4227a7260, 0x1a8099b, 0x2e)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/stores.go:323 +0x100
github.com/cockroachdb/cockroach/pkg/storage.(*Stores).ReadBootstrapInfo(0xc42028fdc0, 0xc4227a7260, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/stores.go:297 +0x79e
github.com/cockroachdb/cockroach/pkg/gossip.(*Gossip).SetStorage(0xc421962180, 0x2637960, 0xc42028fdc0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/gossip/gossip.go:336 +0x126
github.com/cockroachdb/cockroach/pkg/server.(*Node).initStores(0xc420246580, 0x2b23c00d14c0, 0xc42306e1b0, 0xc422d066f0, 0x1, 0x1, 0xc422ba02d0, 0xc422cb0101, 0x0, 0xc42098a620)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:483 +0x842
github.com/cockroachdb/cockroach/pkg/server.(*Node).start(0xc420246580, 0x2b23c00d14c0, 0xc42306e1b0, 0x2637b60, 0xc4204b10a0, 0xc422d066f0, 0x1, 0x1, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:374 +0x411
github.com/cockroachdb/cockroach/pkg/server.(*Server).Start(0xc42174d680, 0x2b23c00d14c0, 0xc42306e1b0, 0x2b23c00d0460, 0xc420090ca0)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:601 +0xdcc
github.com/cockroachdb/cockroach/pkg/server.(*TestServer).Start(0xc42152b830, 0x2630ea0, 0xc4204bf180, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/testserver.go:291 +0x184
github.com/cockroachdb/cockroach/pkg/testutils/serverutils.StartServerRaw(0x2630ea0, 0xc4204bf180, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/testutils/serverutils/test_server_shim.go:164 +0x11a
github.com/cockroachdb/cockroach/pkg/testutils/serverutils.StartServer(0x264e800, 0xc4224d0b40, 0x2630ea0, 0xc4204bf180, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/testutils/serverutils/test_server_shim.go:134 +0x55
github.com/cockroachdb/cockroach/pkg/sql_test.TestParseSentinelValueWithNewColumnInSentinelFamily(0xc4224d0b40)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/schema_changer_test.go:1447 +0x11e
testing.tRunner(0xc4224d0b40, 0x1c572f0)
	/usr/local/go/src/testing/testing.go:610 +0x81
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:646 +0x2ec

goroutine 216747 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*Store).startCoalescedHeartbeatsLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:3441 +0x167
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc422ba02d0, 0xc4207e29d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:209 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x66
--- FAIL: testrace/TestStatusLocalLogs (0.000s)
	status_test.go:211: Get https://127.0.0.1:42462/_status/logfiles/local: net/http: request canceled (Client.Timeout exceeded while awaiting headers)

```
Please assign, take a look and update the issue accordingly.

--- a/pkg/cli/interactive_tests/common.tcl
+++ b/pkg/cli/interactive_tests/common.tcl
@@ -32,12 +32,6 @@ proc interrupt {} {
     sleep 0.4
 }
 
-# Tests that wish to concatenate a marker with the first line of
-# output need to exclude messages about the env. var
-# PROPOSER_EVALUATED_KV as this may be emitted before the logging
-# flags are handled.
-set silence_prop_eval_kv "2>&1 | grep -v 'running with experimental support for proposer-evaluated KV'"
-
 # Convenience functions to start/shutdown the server.
 # Preserves the invariant that the server's PID is saved
 # in `server_pid`.

--- a/pkg/cli/interactive_tests/test_missing_log_output.tcl
+++ b/pkg/cli/interactive_tests/test_missing_log_output.tcl
@@ -37,7 +37,7 @@ eexpect ":/# "
 
 # Check that --alsologtostderr can override the threshold
 # regardless of what --logtostderr has set.
-send "echo marker; $argv start --alsologtostderr=ERROR $silence_prop_eval_kv\r"
+send "echo marker; $argv start --alsologtostderr=ERROR\r"
 eexpect "marker\r\nCockroachDB node starting"
 
 # Stop it.
@@ -45,7 +45,7 @@ interrupt
 interrupt
 eexpect ":/# "
 
-send "echo marker; $argv start --alsologtostderr=ERROR --logtostderr $silence_prop_eval_kv\r"
+send "echo marker; $argv start --alsologtostderr=ERROR --logtostderr\r"
 eexpect "marker\r\nCockroachDB node starting"
 
 # Stop it.
@@ -53,7 +53,7 @@ interrupt
 interrupt
 eexpect ":/# "
 
-send "echo marker; $argv start --alsologtostderr=ERROR --logtostderr=false $silence_prop_eval_kv\r"
+send "echo marker; $argv start --alsologtostderr=ERROR --logtostderr=false\r"
 eexpect "marker\r\nCockroachDB node starting"
 
 # Stop it.
@@ -67,7 +67,7 @@ send "$argv start >/dev/null 2>&1 & sleep 1\r"
 # Now test `quit` as non-start command, and test that `quit` does not
 # emit logging output between the point the command starts until it
 # prints the final ok message.
-send "echo marker; $argv quit $silence_prop_eval_kv\r"
+send "echo marker; $argv quit\r"
 set timeout 20
 eexpect "marker\r\nok\r\n:/# "
 set timeout 5
@@ -76,7 +76,7 @@ set timeout 5
 # that the default logging level is WARNING, so that no INFO messages
 # are printed between the marker and the (first line) error message
 # from quit. Quit will error out because the server is already stopped.
-send "echo marker; $argv quit --logtostderr $silence_prop_eval_kv\r"
+send "echo marker; $argv quit --logtostderr\r"
 eexpect "marker\r\nError"
 eexpect ":/# "
 

--- a/pkg/cli/interactive_tests/test_pretty.tcl
+++ b/pkg/cli/interactive_tests/test_pretty.tcl
@@ -18,7 +18,7 @@ eexpect ":/# "
 
 # Check that tables are not pretty-printed when input is not a terminal
 # and --format=pretty is not speciifed.
-send "echo begin; echo 'select 1;' | $argv sql $silence_prop_eval_kv\r"
+send "echo begin; echo 'select 1;' | $argv sql\r"
 eexpect "begin\r\n1 row\r\n1\r\n1\r\n"
 
 # Check that tables are pretty-printed when input is a terminal

--- a/pkg/storage/replica.go
+++ b/pkg/storage/replica.go
@@ -92,13 +92,7 @@ var txnAutoGC = true
 var tickQuiesced = envutil.EnvOrDefaultBool("COCKROACH_TICK_QUIESCED", true)
 
 // Whether to enable experimental support for proposer-evaluated KV.
-var propEvalKV = func() bool {
-	enabled := envutil.EnvOrDefaultBool("COCKROACH_PROPOSER_EVALUATED_KV", false)
-	if enabled {
-		log.Warningf(context.Background(), "running with experimental support for proposer-evaluated KV, see #10431")
-	}
-	return enabled
-}()
+var propEvalKV = envutil.EnvOrDefaultBool("COCKROACH_PROPOSER_EVALUATED_KV", false)
 
 // ProposerEvaluatedKVEnabled returns whether experimental support for
 // proposer-evaluated KV is enabled.

